docs/usage/usage.md

```diff
@@ -326,6 +337,17 @@ Visualize clusters using one of the embedding methods passing color='leiden'.
     plot.rank_features_groups_tracksplot
 ```
 
+### Data Analysis (Linear Regression, Survival Analysis)
+
+```{eval-rst}
+.. autosummary::
+    :toctree: tools
+    :nosignatures:
+
+    plot.ols_plot
```

Comment:
Think that the function being in the plotting module already suffices to label this as a `plot`.

I would suggest to rename the ols_plot to ols and the kmf_plot to kmf as well.

Reply:
No problem